VimeoUpload/Descriptor System/ProgressDescriptor.swift

```diff
@@ -26,7 +26,7 @@
 
 import Foundation
 
-public class ProgressDescriptor: Descriptor
+open class ProgressDescriptor: Descriptor
```

Comment:
`ProgressDescriptor` is being subclassed by `UploadDescriptor` and `OldUploadDescriptor`, so we should keep it `open` for now.